<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_message">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>command line</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Close">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>210</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>python</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>341</width>
     <height>191</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabAdd">
    <attribute name="title">
     <string>Add</string>
    </attribute>
    <widget class="QTextEdit" name="cliAdd">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>321</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pyAdd">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabDel">
    <attribute name="title">
     <string>Del</string>
    </attribute>
    <widget class="QPushButton" name="pyDel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Del</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="cliDel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>321</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
